src/main/java/pl/tomaszdziurko/jvm_bloggers/UTMParametrsConstants.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+/**
+ * UTM (Urchin Traffic Monitor) URL query parameter identifiers.
+ * 
+ * @author cslysy <jakub.sprega@gmail.com>
+ */
+public class UTMParametrsConstants {
+    
+    /**
+     * Name of the source (usually the domain of source website).
```

Comment:
Instead of this comments, I would only add a link to some website describing UTM at javadoc for class and that would be sufficient as a reference.